omni-relayer/src/workers/near.rs

```diff
@@ -56,7 +137,7 @@ pub async fn sign_transfer(
                             .sign_transfer(
                                 transfer_message.origin_nonce.into(),
                                 Some(config.near.token_locker_id),
-                                transfer_message.fee.into(),
+                                transfer_message.fee.fee.into(),
```

Comment:
We should provide the full fee here. We might need to fix the rainbow-bridge-sdk first to do this.

https://github.com/Near-One/omni-bridge/blob/main/near/nep141-locker/src/lib.rs#L365

Reply:
Will be fixed after merging https://github.com/Near-One/rainbow-bridge-sdk-rs/pull/39

Reply:
Fixed in 77a35c0 (still changes aren't merged in bridge-sdk, but it works locally)